Lib/pyclbr.py

```diff
@@ -52,40 +53,41 @@
 
 class _Object:
     "Information about Python class or function."
-    def __init__(self, module, name, file, lineno, parent):
+    def __init__(self, module, name, file, lineno, end_lineno, parent):
         self.module = module
         self.name = name
         self.file = file
         self.lineno = lineno
+        self.end_lineno = end_lineno
         self.parent = parent
         self.children = {}
         if parent is not None:
             parent.children[name] = self
 
 class Function(_Object):
     "Information about a Python function, including methods."
-    def __init__(self, module, name, file, lineno, parent=None, is_async=False):
-        super().__init__(module, name, file, lineno, parent)
+    def __init__(self, module, name, file, lineno, end_lineno, parent=None, is_async=False):
```

Comment:
Here is the change Guido and I (reluctantly) agreed on.
```suggestion
    def __init__(self, module, name, file, lineno, parent=None, is_async=False, *,  end_lineno=None):
```
Same change for Class.